security/pfSense-pkg-tinc/files/usr/local/pkg/tinc.inc

```diff
@@ -126,7 +126,7 @@ function tinc_save() {
 	file_put_contents("{$configpath}/rsa_key.priv", base64_decode($tincconf['cert_key']) . "\n");
 	chmod("{$configpath}/rsa_key.priv", 0600);
 	if ($tincconf['tinc_up']) {
-		$_output = base64_decode($tincconf['tinc_up']) . "\n";
+		$_output = str_replace("\r", "", base64_decode($tincconf['tinc_up'])) . "\n";
```

Comment:
Might be better to use `unixnewlines()` here instead manually picking what to remove.

Reply:
The code in this file is full of `str_replace("\r","",...)`, I tried to stay consistent with the existing code. However, I agree, this is something that could be improved, however I don't know the the function you mentioned.

Reply:
@buergi `unixnewlines()` can be found at https://github.com/pfsense/pfsense/blob/master/src/etc/inc/util.inc#L2870